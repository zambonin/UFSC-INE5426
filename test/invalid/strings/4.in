# improperly closing/opening simple/double quotes
char a = 'a"
char b = 's
