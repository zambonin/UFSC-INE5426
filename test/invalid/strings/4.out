[Line 2] syntax error, unexpected ID
[Line 3] syntax error, unexpected ID
